magefiles/helm.go

```diff
@@ -145,9 +147,13 @@ func updateChartValue(key, value string) {
 	}
 
 	//// write to file
-	newValueFile, err := yaml.Marshal(&n)
+	var b bytes.Buffer
+	yamlEncoder := yaml.NewEncoder(&b)
+	yamlEncoder.SetIndent(2)
+	yamlEncoder.Encode(&n)
+	err = yamlEncoder.Encode(&n)
 	CheckIfError(err, "HELM Could not Marshal new Values file")
-	err = os.WriteFile(HelmChartValues, newValueFile, 0644)
+	err = ioutil.WriteFile(HelmChartValues, b.Bytes(), 0644)
```

Comment:
ioutil is deprecated: https://pkg.go.dev/io/ioutil

Please use os.WriteFile

Reply:
Ha, yes, done